src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -512,6 +512,26 @@ namespace
         return false;
     }
 
+    bool get_photon_target(Object* object, const TimeValue time)
+    {
+        if (object->SuperClassID() == GEN_DERIVOB_CLASS_ID)
+        {
+            IDerivedObject* derived_object = static_cast<IDerivedObject*>(object);
+            for (int i = 0, e = derived_object->NumModifiers(); i < e; ++i)
+            {
+                Modifier* modifier = derived_object->GetModifier(i);
+                if (modifier->ClassID() == AppleseedObjPropsMod::get_class_id())
+                {
+                    int photon_target = 0;
```

Comment:
This is correct, photon_target is an integer initialized to 0. 
The UI Boolean switch passes an integer #define TRUE 1 or #define FALSE 0
which is compared to photon_target and returns a Boolean:
`return optimize_for_instancing == TRUE;`

Reply:
> Can you confirm that the photon target flag is properly saved in projects?

The settings are saved. I used the same method the `Optimize for Instancing` flag has been implemented.

Reply:
> This is correct, photon_target is an integer initialized to 0.
> The UI Boolean switch passes an integer #define TRUE 1 or #define FALSE 0

Exactly. What I meant is that you should use `FALSE` instead of 0, for clarity.